### PR TITLE
move section logic out of Tiles and to responsibility of caller

### PR DESCRIPTION
**Pull request by nickjvm**
First pass; open to conversation regarding implementation.

### CODE REVIEW

File: src/js/components/Tiles.js [hunk 1]
```diff
@@ -112,71 +112,34 @@ export default class IndexTiles extends Component {
   }
 
   _renderSections (classes, onMore) {
-    const { result, selection, sort } = this.props;
-    const parts = sort.split(':');
-    const attributeName = parts[0];
-    const direction = parts[1];
+    const { result } = this.props;
     let sections = [];
-    let items = result.items.slice(0);
```
Comment (alansouzati) [inline src/js/components/Tiles.js:120]: I believe we can safely use `const` for sections.

**Comment by alansouzati** [comment]
for me good to go this minor issue can be resolved either now or later.

**Comment by nickjvm** [comment]
Thanks! @ericsoderberghp, any feedback?

**Comment by nickjvm** [comment]
@alansouzati @ericsoderberghp I'm going to close this one for now and reopen later. We're using the stable branch for bulk action changes from last week and don't want to risk this being on stable and breaking our demo tomorrow.

**Comment by nickjvm** [comment]
Looks like I have some merge conflicts to take a look at.

**Pull request closed without merging**